src/verifact_manager.py

```
@@ -118,19 +119,29 @@
 
     async def _detect_claims(self, text: str) -> list[Claim]:
         logger.info("Detecting claims...")
-        result = await Runner.run(claim_detector_agent, text)
-
-        claims = result.final_output_as(list[Claim])
+        claims = await self.claim_detector.process(text)
         logger.info(f"Detected {len(claims)} claims")
 
         return claims
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)